.circleci/config.yml

```diff
@@ -77,11 +97,91 @@ jobs:
                 fi
             done
 
+  build-and-publish-single-chart:
+    docker:
+      - image: banzaicloud/helm:0.0.2
+
+    working_directory: /workspace/banzai-charts
+
+    steps:
+      - checkout
+
+      - run:
+          name: Setup helm repositories
+          command: |
+            helm init -c
+            helm repo remove local
+            helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator
```

Comment:
(why) do we need this?

Reply:
https://github.com/helm/charts/pull/12372/files

Reply:
Required for `cadence`
```bash
$ helm dependency update cadence/
Error: no repository definition for http://storage.googleapis.com/kubernetes-charts-incubator. Please add them via 'helm repo add'
```